app/models/APIModel.scala

```diff
@@ -1,11 +1,5 @@
 package models
 
 case class Status(service: String, environment: String, serverTime: String) {
-  
-}
-
-
-
-
-
 
+}
```

Comment:
another thing, the brackets here are redundant. you are not declaring any associated functions to the case class, so you can delete them